packages/rocketchat-ui/client/views/app/createRooms.js

```diff
@@ -0,0 +1,49 @@
+const roomTypesBeforeStandard = function() {
+	const orderLow = RocketChat.roomTypes.roomTypesOrder.filter((roomTypeOrder) => roomTypeOrder.identifier === 'c')[0].order;
+	return RocketChat.roomTypes.roomTypesOrder.filter(
+		(roomTypeOrder) => roomTypeOrder.order < orderLow
+	).map(
+		(roomTypeOrder) => {
+			return RocketChat.roomTypes.roomTypes[roomTypeOrder.identifier];
+		}
+	).filter((roomType) => roomType.creationTemplate && roomType.canBeCreated());
+};
+
+const roomTypesAfterStandard = function() {
+	const orderHigh = RocketChat.roomTypes.roomTypesOrder.filter((roomTypeOrder) => roomTypeOrder.identifier === 'd')[0].order;
+	return RocketChat.roomTypes.roomTypesOrder.filter(
+		(roomTypeOrder) => roomTypeOrder.order > orderHigh
+	).map(
+		(roomTypeOrder) => {
+			return RocketChat.roomTypes.roomTypes[roomTypeOrder.identifier];
+		}
+	).filter((roomType) => roomType.creationTemplate && roomType.canBeCreated());
```

Comment:
@geekgonecrazy There are actually two ways of preventing the room type from creating a tab: Only those with a creation template are being included and for each of them, the validation for the permissions is performed.

Reply:
Perfect :+1: